### DISCUSSION TITLE
One way roads

### DISCUSSION THREAD

**Discussion by zeddash** [voxeltycoon/issues#340] [General]
It's often when having an array of terminals if all trucks follow a designated route you are able to eliminate trucks crossing paths completely which reduces stops (only when merging).

This would also offer up the ability to use both lanes to travel in the same direction and possibly allowing overtaking - but this isn't important, I would like to just mark a lane as to be left empty unless there is no other route.

I'm imagining this as the same roads in the game currently with one lane with a yellow box pattern on it marking it to not be used unless necessary - a single lane road would be useful to get an extra square of space I don't feel it is necessary and leaving the lane allows for easier replacement with a 2 lane road (I play cities skylines and changing road widths is a pain)

Equally just let us change the junction connections to not let a truck turn left or right to force them to go on the correct roads.

**Comment by victorleblais** [comment]
Planned feature:
![image](https://user-images.githubusercontent.com/4365075/115476587-73d7dd00-a242-11eb-8ed3-19adbe2a2633.png)

https://voxeltycoon.xyz/devlog/ea-faq/
